### ISSUE TITLE
Transparent canvas - iOS 11.3 beta

### ISSUE THREAD

**Issue by horyzont** [pixijs/pixi.js#4660]
There is a problem with pixi 4.7.0 and transparent parameter.
Bug is reproducible on iPhone SE/6S/7 and iOS 11.3 beta from this link: https://jsfiddle.net/b3ctuuez/.
It is not reproducible on Chrome/Firefox/IE.

**Comment by bigtimebuddy** [comment]
Are other iPhones or iOS devices affected aside from the ones you mentioned? Or are those only ones you've tested? Does transparent work correctly on older version of PixiJS? 

Bug does not exist on latest iOS 11.2.5 on iPhone 6s.

If you can create a simple WebGL canvas (no PixiJS) and it doesn't respect transparently, then likely, this is a Safari/WebKit bug and should be filed with Apple.

**Comment by macguffin** [comment]
I have just noticed the same behaviour. @horyzont if you report this to Apple could you post a link to the bug so I can follow the progress. 
Many thanks

**Comment by horyzont** [comment]
Hope I've reported it in proper place: https://bugreport.apple.com/web/?problemID=37234491

**Comment by macguffin** [comment]
Thanks for posting but I think you can only see that as it's probably linked to your dev account.
Would really appreciate if you posted any feedback/movement from apple.

**Comment by bigtimebuddy** [comment]
Thanks @horyzont for digging into this.

**Comment by ivanpopelyshev** [comment]
Workaround:

```js
var app = new PIXI.Application(800, 600, {backgroundColor: 0x1099bb, transparent: 'notMultiplied'});
app.renderer._backgroundColorRgba[3] = 1.0;
```

**Comment by ivanpopelyshev** [comment]
According to [forum post](http://www.html5gamedevs.com/topic/35573-pixi-canvas-is-coming-white-on-mac-safari/) , Mac 10.12.6, Safari 11.0.2 (12604.4.7.1.6) has the same issue. It is not mobile-only.

**Comment by horyzont** [comment]
This workaround:

```
var app = new PIXI.Application(800, 600, {backgroundColor: 0x1099bb, transparent: 'notMultiplied'});
app.renderer._backgroundColorRgba[3] = 1.0;
```

doesn't work. Background is still not transparent. You can verify here: https://jsfiddle.net/7w59pmmm/

**Comment by ivanpopelyshev** [comment]
Its workaround for not transparent background. Sorry, no transparent for that safari version.

**Comment by macguffin** [comment]
Good news, looks like this is fixed in 11.3 beta 3 :)

**Comment by horyzont** [comment]
Excellent 👍 We will check it soon.. Thanks!
